src/app/login/login.component.ts

```diff
@@ -28,6 +40,10 @@ export class LoginComponent implements OnDestroy {
             });
     }
 
+    loginFb() {
+        this.windowRef.nativeWindow.location.href = this.fbLoginUrl;
```

Comment:
I am not fan how its made, you can use facebook graph api instead, nevertheless such implementation details should be encapsulated in service like ```FacebookAuthService```